managed_nagios_plugin/nagios/tasks.py

```diff
@@ -571,7 +571,9 @@ def configure(ctx):
 @operation
 def start(ctx):
     ctx.logger.info('Enabling and starting nagios and httpd services')
-    services = ['nagios', 'httpd', 'nagiosrest-gunicorn', 'incrond']
+    services = ['nagios', 'incrond']
+    if ctx.node.properties['start_nagiosrest']:
+        services.append(['nagios', 'httpd'])
```

Comment:
I think this is meant to be nagiosrest-gunicorn and httpd?

Also, you need to use services.extend here or you'll end up with a nested list.

Additionally, I see the same list is used later in the file, so maybe factor this out into a NAGIOSREST_SERVICES constant.

Reply:
Sure I'll fix that